Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,21 @@
+#include "BatteryData.h"
 #include "BatteryStateOfChargeService.h"
-
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const int magicNumber = 3600000;
```

Comment:
This is a magic number, but it shouldn't be called a magic number. What does this value do or represent?